.idea/codeStyles/Project.xml

```diff
@@ -47,31 +47,9 @@
       <option name="SPACE_BEFORE_SUPERCLASS_COLON" value="false" />
     </Objective-C>
     <Objective-C-extensions>
```

Comment:
Is it required to keep the section or can it be removed completely?

Reply:
You cannot disable a codestyle section 
![image](https://user-images.githubusercontent.com/3358282/76951011-e02b3300-6902-11ea-8cc1-f6fa8eb32568.png)

so yeah, required to keep

Reply:
In that case we should keep it empty for all other languages too.